pages/common/go-get.md

```diff
@@ -0,0 +1,16 @@
+# go get
+
+> Add a dependency package, or download packages in legacy GOPATH mode.
+> More information: <https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them>.
+
+- Add a specified package to go.mod in module-mode or install the package in GOPATH-mode:
+
+`go get {{example.com/pkg}}`
+
+- In module-mode, modify the package with a given version:
```

Comment:
This sounds like you have to enter `module-mode` first. From what I read in the docs, what do you think about
```suggestion
- Modify the package with a given version in module-aware mode:
```
This also applies to the last example